### PR TITLE
Feat : Updated blog component

### PR DESCRIPTION
**Pull request by cp-ashish-k**
- change the strapi url in config.js file with the admin url
- Remove canopas from nuxt.config.js file it should be  "@canopassoftware/blog-components"

### CODE REVIEW

File: nuxt-frontend/config.js [hunk 1]
```diff
@@ -135,6 +135,22 @@ export default Object.freeze({
     url: "https://dev-stack.canopas.com/frontend-development",
     image: "https://dev-stack.canopas.com/apple-touch-icon.png",
   },
+
+  // For blog components
+  STRAPI_URL: "https://bq465hjg3t-admin.canopas.com",
```
Comment (cp-sumi-k) [inline nuxt-frontend/config.js:140]: Let's set it from env var

File: nuxt-frontend/config.js [hunk 2]
```diff
@@ -135,6 +135,22 @@ export default Object.freeze({
...
+  NOT_FOUND: 404,
+  WORDS_PER_MINUTE: 200,
+  SUCCESS: 200,
```
Comment (cp-sumi-k) [inline nuxt-frontend/config.js:143]: If it not required, let's remove it

File: nuxt-frontend/config.js [hunk 3]
```diff
@@ -135,6 +135,22 @@ export default Object.freeze({
...
+  SERVER_ERROR: 500,
+  COMPANY_TITLE: "Canopas",
+  COMPANY_NAME: "Canopas Software LLP.",
+  BLOG_SEO_META_DATA: {
```
Comment (cp-sumi-k) [inline nuxt-frontend/config.js:147]: Let's write code lines in consistent way for better readability.

File: nuxt-frontend/pages/[slug].vue [hunk 4]
```diff
@@ -4,23 +4,145 @@
...
+const recommandedBlog = computed(() => recStore.items);
+
+let published_time, recommandedPosts;
+await useAsyncData("blog", () => store.loadResource(slug.value));
+if (status.value !== config.SUCCESS) {
+  emit("notfound");
+} else {
+  post.value = postData?.value;
+}
+published_time = new Date(post?.value?.publishedAt).toLocaleTimeString();
```
Comment (cp-sumi-k) [inline nuxt-frontend/pages/[slug].vue:62]: `publishedAt` to `published_on`

File: nuxt-frontend/pages/resources/index.vue [hunk 5]
```diff
@@ -4,15 +4,72 @@
...
 
 <script setup>
 import Header from "@/components/partials/NewHeader.vue";
-import config from "./../../config";
+import config from "@/config";
+import { useBlogListStore } from "@/stores/resources";
 
 const { $mixpanel } = useNuxtApp();
+const posts = ref([]);
+let postLimit = 10;
```
Comment (cp-sumi-k) [inline nuxt-frontend/pages/resources/index.vue:28]: let's make `10` as constant

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: nuxt-frontend/pages/[slug].vue [hunk 6]
```diff
@@ -4,23 +4,145 @@
...
+  useRecommandedBlogStore,
+} from "@/stores/resources";
+import { filterPostsByCategoryAndTag } from "@/utils";
 const { $mixpanel } = useNuxtApp();
+const post = ref([]);
 const route = useRoute();
 const slug = ref(route.params.slug);
+const iframelyKey = config.VITE_IFRAMELY_KEY;
+const emit = defineEmits(["notfound"]);
+const socialmediadata = {
```
Comment (cp-sumi-k) [inline nuxt-frontend/pages/[slug].vue:39]: Let's rename it in camel case

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: nuxt-frontend/pages/resources/index.vue (hunk 5 above)
Comment (cp-sumi-k) [reply nuxt-frontend/pages/resources/index.vue]: 10 should be constant like `const POST_LIMIT = 10` not ref

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: nuxt-frontend/pages/resources/index.vue [hunk 7]
```diff
@@ -3,16 +3,73 @@
...
 
 <script setup>
 import Header from "@/components/partials/NewHeader.vue";
-import config from "./../../config";
+import config from "@/config";
+import { useBlogListStore } from "@/stores/resources";
 
 const { $mixpanel } = useNuxtApp();
+const posts = ref([]);
+const postLimit = ref(10);
```
Comment (cp-sumi-k) [inline nuxt-frontend/pages/resources/index.vue:27]: 10 should be constant like `const POST_LIMIT = 10` not ref

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: nuxt-frontend/pages/resources/index.vue (hunk 7 above)
Comment (cp-ashish-k) [reply nuxt-frontend/pages/resources/index.vue:27]: cannot use that as the value needs to be increased as the user reaches the last blog to show more blogs..

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**